aggregator/src/app/BundleService.ts

```diff
@@ -273,6 +326,13 @@ export default class BundleService {
         },
       });
 
+      if (aggregateBundle) {
```

Comment:
I tried a few different places in the code to set the `aggregateBundleHash` and settled on here. Wanted to update it as close to the submission as possible, while still having to do the update once so wouldn't have to set it in `handleFailedRow` and `submitAggregateBundle`.

Reply:
hmmm after writing that, now questioning whether this is the best place. Do we even need to update the `aggregateBundleHash` if the bundle is going to fail? In which case, does just before the line: `await this.submitAggregateBundle(` work better?